src/components/PostList/PostList.tsx

```diff
@@ -1,3 +1,16 @@
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+import { Post } from '../../Types/Posts';
+
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <div className="PostInfo">
+        <PostInfo {...post} />
+      </div>
+    ))}
```

Comment:
```suggestion
      <div className="PostInfo">
        <PostInfo {...post} />
      </div>
    ))}
```

на перший відкриваючий тег потрібно додати атрибут key=""